snippets/csharp/VS_Snippets_CLR/conceptual.calendars/cs/instantiatewithera1.cs

```diff
@@ -28,11 +27,11 @@ public static void Main()
 }
 // The example displays the following output:
 //    Date instantiated without an era:
-//    1/1/2 in Japanese Calendar -> 1/1/1990 in Gregorian
```

Comment:
The change here looks good but it depends on the updates of the era data and also we'll need to change this doc again in the future when getting any new era. 

Can we just change the part at the end to tell something can explain it? I mean instead of having 

// The example displays the following output:
// 1/1/2 in Japanese Calendar -> 1/1/2020 in Gregorian

maybe we can say, the code sample will show the equivalent Gregorian date that match year 2 month 1 day 1 of the current era on the system.